### PR TITLE
Support timestamp format option

### PR DESCRIPTION
**Pull request by kamatama41**
This PR makes display format of log's timestamp configurable by `--timestamp-format` option.

Originally my motivation is to remove the timestamp so I will use the option like `--timestamp-format=''`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ env:
   - GO111MODULE=on
 
 go:
-  - 1.12.x
-  - 1.13.x
+  - 1.15.x
```
Comment (kamatama41) [inline .travis.yml:7]: Updated to the versions that are currently supported

File: cmd/run.go [hunk 2]
```diff
@@ -36,6 +37,7 @@ func runTaskCmd() *cobra.Command {
 	flags.StringVarP(&r.securityGroups, "security-groups", "g", "", "Provide security group IDs with comma-separated string (sg-0123asdb,sg-2345asdf), if you want to attach the security groups to ENI of the task.")
 	flags.BoolVarP(&r.fargate, "fargate", "f", false, "Whether run task with FARGATE")
 	flags.IntVarP(&r.timeout, "timeout", "t", 0, "Timeout seconds")
+	flags.StringVarP(&r.timestampFormat, "timestamp-format", "", "[2006-01-02 15:04:05.999999999 -0700 MST]", "Format of timestamp for outputs. You should follow the style of Time.Format (see https://golang.org/pkg/time/#pkg-constants)")
```
Comment (kamatama41) [inline cmd/run.go:40]: The default value is the format of `Time.String`
https://github.com/golang/go/blob/go1.16/src/time/format.go#L451

**Comment by kamatama41** [comment]
@h3poteto 
Cloud you take a look at this? thanks.

**Review by h3poteto — APPROVED** [review]

**Comment by h3poteto** [comment]
Thank you!

**Comment by kamatama41** [comment]
Thank you very much for the quick review 😃

**Pull request merged**